app/src/controllers/peach.ts

```diff
@@ -38,7 +42,11 @@ const PEACH_TRACKING_PRIORITY = [
 export const getPeachRecordController = async (req: Request<{ recordId: string; systemId: string }>, res: Response) => {
   const response = await getPeachRecord(req.params.recordId, req.params.systemId);
 
-  const peachSummary: PeachSummary = summarizeRecord(response);
+  const peachSummary: PeachSummary | null = summarizeRecord(response);
+
+  // TODO-RELEASE: make sure that we properly handle this case, not a 404
```

Comment:
As mentioned in PADS-679, this will remain an HTTP 404, so this todo is not needed.